lib/containers/urls.pm

```diff
@@ -116,3 +117,25 @@ sub get_suse_container_urls {
 
     return (\@untested_images, \@released_images);
 }
+
+sub get_3rd_party_images {
+    my $ex_reg = get_var('REGISTRY', 'docker.io');
+    my @images = (
+        "registry.opensuse.org/opensuse/leap",
+        "registry.opensuse.org/opensuse/tumbleweed",
+        "$ex_reg/library/alpine",
+        "$ex_reg/library/debian",
+        "$ex_reg/library/ubuntu",
+        "$ex_reg/library/centos",
+        "$ex_reg/library/fedora",
+        "registry.access.redhat.com/ubi8/ubi",
+        "registry.access.redhat.com/ubi8/ubi-minimal",
+        "registry.access.redhat.com/ubi8/ubi-init");
+    push @images, (
+        "registry.access.redhat.com/ubi7/ubi",
+        "registry.access.redhat.com/ubi7/ubi-minimal",
+        "registry.access.redhat.com/ubi7/ubi-init"
+    ) unless (check_var('ARCH', 'aarch64'));
```

Comment:
Now there are arch-specific exclusions both here in `get_3rd_party_images` and in `test_3rd_party_image`.
I suggest to move them here instead and not even mention images which aren't tested.